src/utils/enum.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export const getStringEnumKeys = (E: any) =>
+  Object.keys(E).filter((k) => typeof E[k as any] === 'string');
+
+export const getStringEnumValues = (E: any) =>
+  getStringEnumKeys(E).map((k) => E[k as any]);
```

Comment:
Big fan. Pretty cool

Reply:
🚀 
I find it's a bit strange that enums don't have a built-in method for this. Kinda makes me lean towards wanting to use object literals and const assertions.